reactor-netty-http/src/test/java/reactor/netty/BaseHttpTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2011-Present VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty;
+
+import org.junit.jupiter.api.AfterEach;
+import reactor.netty.http.client.HttpClient;
+import reactor.netty.http.server.HttpServer;
+import reactor.netty.resources.ConnectionProvider;
+
+import java.net.SocketAddress;
+import java.util.function.Supplier;
+
+/**
+ * Provides utility methods for {@link HttpServer} and {@link HttpClient} creation.
+ * At the end of each test guarantees that the server is disposed.
+ *
+ * @author Violeta Georgieva
+ * @since 1.0.3
+ */
+public class BaseHttpTest {
+
+	protected DisposableServer disposableServer;
+
+	@AfterEach
+	protected void tearDown() throws Exception {
```

Comment:
consider renaming this method to something more "unique" so that existing `tearDown()` methods don't need to override and call `super.tearDown()`

Reply:
fixed